portugues/pescaria/como_seria.md

```diff
@@ -0,0 +1,7 @@
+E junto com a lembrança vêm a dúvida!!
+
+Onde melhor serianal
+
+[No Pantanal](pantanal/na_canoa.md)
+
+[No Mar da Bahia](no_mar_da_bahia.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.